vaadin-spring-tests/test-spring-security-flow/pom.xml

```diff
@@ -7,8 +7,8 @@
         <artifactId>vaadin-spring-tests</artifactId>
         <version>17.0-SNAPSHOT</version>
     </parent>
-    <artifactId>test-spring-secturity-helper</artifactId>
-    <name>Integration tests for Vaadin Spring Security helper</name>
+    <artifactId>test-spring-secturity-flow</artifactId>
```

Comment:
Noticed a typo in the original artifactId (`secturity`) that perhaps could be fixed in this change as well.

Reply:
Ifxed teh ypto